index.js

```diff
@@ -296,6 +300,10 @@ async function run() {
     const serviceInput = core.getInput('service', { required: createServiceIfNotFound });
     const serviceConfigFile = core.getInput('service-config-file', { required: createServiceIfNotFound });
 
+    if (serviceConfigFile && !createServiceIfNotFound) {
+      throw new Error(`Input 'service-config-file' is required only if 'create-service-if-not-found' is true.`); 
```

Comment:
Acho que como está fica um pouco confuso, invés de "requirido" acho que "permitido" fica mais claro, o que acha?

```suggestion
      throw new Error(`Input 'service-config-file' is allowed only if 'create-service-if-not-found' is true.`); 
```